README.adoc

```diff
@@ -1,69 +1,75 @@
-# Localization Support Plugin
+[[localization-support-plugin]]
+= localization-support-plugin
+:toc: macro
+:toc-title:
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+image:https://img.shields.io/jenkins/plugin/v/localization-support.svg[Jenkins Plugin,link=https://plugins.jenkins.io/localization-support]
+image:https://img.shields.io/jenkins/plugin/i/localization-support.svg?color=blue[Jenkins Plugin Installs,link=https://plugins.jenkins.io/localization-support]
+image:https://ci.jenkins.io/job/Plugins/job/localization-support-plugin/job/master/badge/icon[Build Status,link=https://ci.jenkins.io/job/Plugins/job/localization-support-plugin/job/master/]
```

Comment:
Not a fan of badges that are entirely redundant everywhere they're shown.

```suggestion
```

Reply:
Where else are they shown?

Reply:
Plugin site shows install count and latest version:

> ![Screenshot](https://user-images.githubusercontent.com/1831569/98873471-bf5ab380-2478-11eb-8779-f3953b976f83.png)

GitHub repo shows CI status and latest version once I migrate the changelog over to GH releases:

> ![Screenshot](https://user-images.githubusercontent.com/1831569/98873515-d7cace00-2478-11eb-95a2-eb5b95e897ab.png)